### PR TITLE
Lab3

### PR DESCRIPTION
**Pull request by Sdude225**
Bottom tabs, didn't found API based on online courses, so used one which searches for available job positions. Coroutines, bottom tabs, small ui fixes

### CODE REVIEW

**Review by ASV44 — CHANGES_REQUESTED** [review]
@Sdude225 Add `app/build` directory to git ignore and clean up project version control

**Comment by Sdude225** [comment]
@ASV44 Done

File: app/src/main/java/com/sample/app/feed/FeedActivity.kt [hunk 1]
```diff
@@ -4,44 +4,80 @@ import android.content.Context
...
+//        }
+//    }
+//
+//    private fun initListView() {
+//        verticalList = findViewById<View>(R.id.vertical_list) as RecyclerView
+//        verticalList!!.layoutManager = LinearLayoutManager(this, LinearLayoutManager.VERTICAL, false)
+//        val verticalAdapter = VerticalAdapter()
+//        verticalAdapter.setData(itemList)
+//        verticalList!!.adapter = verticalAdapter
+//    }
```
Comment (ASV44) [inline app/src/main/java/com/sample/app/feed/FeedActivity.kt]: Remove commented code. It does nothing and there is no reason to keep it

**Review by ASV44 — CHANGES_REQUESTED** [review]
@Sdude225 You have conflicts. Rebase current branch and force push after that

**Pull request merged**